src/transaction/deploy_account.rs

```diff
@@ -132,6 +134,20 @@ impl DeployAccount {
         )
     }
 
+    fn constructor_entry_points_empty(
+        &self,
+        contract_class: CompiledClass,
+    ) -> Result<bool, StateError> {
+        match contract_class {
+            CompiledClass::Deprecated(class) => Ok(class
+                .entry_points_by_type
+                .get(&EntryPointType::Constructor)
+                .ok_or(ContractClassError::NoneEntryPointType)?
+                .is_empty()),
+            CompiledClass::Casm(class) => Ok(class.entry_points_by_type.constructor.is_empty()),
```

Comment:
we have repeated this code in several places now, i think it would be a good idea to move it a single place and call it from there for example in `utils.rs` or a related module.